backend/user-service/.env.sample

```diff
@@ -29,7 +29,8 @@ USER=EMAIL_ADDRESS
 PASS=PASSWORD
 
 # Redis configuration
-REDIS_URI=REDIS_URI
+REDIS_URI=redis://redis:6379 # Uncomment if you're running the user service with docker
```

Comment:
i think the first one can say run user service with all the other service using docker compose, and second one run individually? (if i understood correctly)